app/src/main/java/de/tum/in/tumcampusapp/activities/SurveyActivity.java

```diff
@@ -470,22 +537,20 @@ private void setUpTabHost() {
         tabHost.addTab(tabSpec);
     }
 
-    //check if edittext is empty
-    private boolean hasQuestion(EditText et) {
-        return !et.getText().toString().isEmpty();
-    }
 
-    //check if user is allowed to submit survey depending on the last survey date and number of question he submitted before
+    /**
+     * check if user is allowed to submit survey depending on the last survey date and number of question he submitted in the last week
+     */
     @SuppressLint("SetTextI18n")
-    private void userAllowed() {
+    private void numberOfQuestionsUserAllowed() {
```

Comment:
merged with another function.
setUpSpinnerForQuestionsNumber()
//checks available number of questions and handles numberOfquestions spinner liseneter